tests/framework/di/InstanceTest.php

```diff
@@ -46,6 +46,26 @@ public function testEnsure()
         $this->assertTrue(Instance::ensure(['class' => 'yii\db\Connection', 'dsn' => 'test'], 'yii\db\Connection', $container) instanceof Connection);
     }
 
+    /**
+     * ensure an InvalidConfigException is thrown when a component does not exist.
+     */
+    public function testEnsureNonExistingComponent()
```

Comment:
therefore I added the comment, it would be redundant to try to encode the whole description from phpdoc in the method name.